dsgrid/config/mapping_tables.py

```diff
@@ -130,12 +130,21 @@ def add_records(cls, records, values):
 
         return records
 
-    def dict(self, by_alias=True, exclude=None):
-        if exclude is None:
-            exclude = set()
-        exclude.add("records")
-        data = super().dict(by_alias=by_alias, exclude=exclude)
-        return serialize_model_data(data)
+    def dict(self, *args, **kwargs):
+        exclude = {"records"}
+        if "exclude" in kwargs and kwargs["exclude"] is not None:
+            kwargs["exclude"].union(exclude)
+        else:
+            kwargs["exclude"] = exclude
```

Comment:
These lines of code are repeated a lot in this PR. Is that okay/should it be refactored?

Reply:
Fixed